Include requirements around grant communication (#52)
use-inkubator/Ecosystem-Grants: README.md

- Your team has **proven experience** with relevant languages, technologies, and/or a strong technical background. GitHub profiles of team members should be included in your application.
- Your application is **rich in technical details** and well-defined.

Additionally, all projects must fulfill the following requirements

- Your project uses the RISC-V compatible version of the ink! language, [v6.X releases](https://use.ink/6.x/). The current cohort is focused on funding developments willing to push the ecosystem forward by leveraging ink!'s stack in its most ecosystem-aligned version.
- All code produced must be **open-sourced**, without relying on closed-source software for full functionality. Preferred licenses include Apache 2.0, GPLv3, MIT, or Unlicense.
- Projects involved in token sales are not eligible.
- Applications must not mention a specific token. Instead, focus on the software being implemented/researched.
- Teams must finish an ongoing grant before applying for another one.
- We do not fund projects encouraging gambling, illicit trade, money laundering, or criminal activities.
- Communication:

>   - After you got the grant create a post on the Polkadot Forum. Post regular updates in that thread until the grant is finished.

Reviewer: ```suggestion
  - After your grant application is approved, create a post on the Polkadot Forum. Post regular updates in that thread until the grant is finished.
```
